packages/ozone/src/mod-service/index.ts

```diff
@@ -139,6 +147,20 @@ export class ModerationService {
     if (createdBy) {
       builder = builder.where('createdBy', '=', createdBy)
     }
+    if (createdAfter) {
+      builder = builder.where('createdAt', '>=', createdAfter)
+    }
+    if (createdBefore) {
+      builder = builder.where('createdAt', '<=', createdBefore)
+    }
+    if (hasComment) {
+      builder = builder.where((qb) => {
+        if (commentKeyword) {
+          return qb.where('comment', 'ilike', `%${commentKeyword}%`)
+        }
+        return qb.where('comment', 'is not', null)
+      })
+    }
```

Comment:
Would there be any issue making `hasComment` and `commentKeyword` independent from each other?

Reply:
nope, no issues at all.